### ISSUE TITLE
Convert manual PP to websocket

### ISSUE THREAD

**Issue by OmgImAlexis** [pymedusa/Medusa#3885]
Based on issues like https://github.com/pymedusa/Medusa/issues/3878#issuecomment-374881601 I think it's worth switching the manual PP process from a submission to an ajax call similar to the manual search page.

**Comment by OmgImAlexis** [comment]
Added to https://github.com/pymedusa/Medusa/issues/282

**Comment by p0psicles** [comment]
Yeah it's on the todo. Eventually I want to add a postprocessing thread, that queues a postprocess action.
And the queue gives back an id/reference. The reference can be used to check on the postprocess state.
This would also be used for external scripts like nzbToMedia. But then we should PR changes to scripts like nzbToMedia, to have it check async like it does for example with cp.
